index.html

```diff
@@ -0,0 +1,328 @@
+                    <h1>My Recent Works</h1>
+                </div>
+            </div>
+            
+            <div class="grid-item item1">
+                <div class="img-holder">
+                    <img class="work-banner" src="/assets/images/Img Placeholder.png" alt="workimage"> 
+                </div>
+                <div class='text-group'>
+                    <hgroup>
+                        <h1 class="work_primary_text">Multi-Post Stories</h1>
+                        <p>
+                        A daily selection of privately personalized
+                        reads; no accounts or sign-ups required.This 
+                        has been the industry's standard dummy text 
+                        ever since the 1500s, when an uknown printer
+                        took a standaed dummy text
+                        </p>
+                    </hgroup>
+                    <div id="languages" class="languages">
+                        <ul class="language">
+                            <li> <a href="#">CSS</a></li>
+                            <li> <a href="#">HTML</a></li>
+                            <li> <a href="#">Bootstrap</a></li>
+                            <li> <a href="#">Ruby</a></li>
+                        </ul>
+                    </div>
+                    <div class="btn-holder">
+                        <button class="btn">See Project</button>
+                    </div>
+                </div>
+            </div>
+            <div class="grid-item item2">
+                <div class="content">
+                    <h1 class="p-text">Professional Art <br>
+                        Printing Data
+                    </h1>
+                    <h2 class="s-text">
+                        A daily selection of privately
+                        personalized reads; no accounts or
+                        sign-ups required. Has been the 
+                        industry's standard.
+                    </h2>
+                    <div class="tags">
+                        <ul class="tag">
+                            <li> <a href="#">HTML</a></li>
+                            <li> <a href="#">Bootstrap</a></li>
+                            <li> <a href="#">Ruby</a></li>
+                        </ul>
+                    </div>
+
+                    <div class="btn2-holder">
+                        <a href="#" class="btn2">See Projects</a>
+                    </div>
+                </div>
+            </div>
+            <div class="grid-item item2">
+                <div class="content">
+                    <h1 class="p-text">Professional Art <br>
+                        Printing Data
+                    </h1>
+                    <h2 class="s-text">
+                        A daily selection of privately
+                        personalized reads; no accounts or
+                        sign-ups required. Has been the 
+                        industry's standard.
+                    </h2>
```

Comment:
> - [ ] I suggest you use a p tag instead of a heading tag. This way will be semantically correct and the screen readers and other technologies will thank you.😊